### ISSUE TITLE
Factors affecting test results

### ISSUE THREAD

**Issue by BinLi-git** [WebKit/MotionMark#56]
Hi, Is the final test result related to network factors? Why do the test results fluctuate so much in several tests?

**Comment by smfr** [comment]
No, all the content is already loaded when the test runs. Various things affect the level of noise in the results; these can be inspected if you run from `developer.html` and look at the graphs at the end of the run.
